probatus/metric_volatility/volatility.py

```diff
@@ -99,8 +101,9 @@ def __init__(self, model, scoring='roc_auc', test_prc=0.25, n_jobs=1, stats_test
 
         self.stats_tests_objects = []
         if len(self.stats_tests_to_apply) > 0:
```

Comment:
This condition will never be met if you just set the list to be empty. Why is this code here? At very least add comment

Reply:
Correction: I misread, sorry! the suffix is different.